app/data_model/database.py

```diff
@@ -19,23 +19,28 @@
 class QuestionnaireState(base):
     __tablename__ = 'questionnaire_state'
     user_id = Column('userid', String, primary_key=True)
-    state = Column('questionnaire_data', String)
+    _state = Column('questionnaire_data', String)
+    _data = None
 
     def __init__(self, user_id, data):
         self.user_id = user_id
-        self.state = json.dumps(data)
+        self._data = data
+        self._state = json.dumps(self._data)
```

Comment:
This isn't Private with a capital "P", it's just a Python convention that indicates it's not a good idea to access/assign to it directly.